.github/workflows/CI.yaml

```diff
@@ -96,7 +92,8 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_TEST_SECRET }}
         run: |
           export TRAVIS=true
-          pytest -v --cov-report xml --cov=perses --durations=0 -a "not advanced" -n auto -m "not gpu_needed" perses/tests
+          # TODO: revert this back to testing the entire test suite when finished with #1177
+          pytest -v --cov-report xml --cov=perses --durations=0 -a "not advanced" -n auto -m "not gpu_needed" perses/tests/protocols
```

Comment:
Do we want to keep this for now or revert it? I guess other PRs opened agains #1066 could benefit from having this as it is right now.

Reply:
Hmmm, I'm fine with leaving it, since we can't merge #1066 without fixing the merge conflicts anyway. Low risk it makes it through merge.